src/vs/base/common/observableInternal/logging.ts

```diff
@@ -105,6 +105,28 @@ export class ConsoleObservableLogger implements IObservableLogger {
 			this.changedObservablesSets.get(derived)!.add(observable);
 			return existingHandleChange.apply(derived, [observable, change]);
 		};
+
+		const debugTrackUpdating = false;
```

Comment:
Should this live with the other debug value here?

https://github.com/microsoft/vscode/blob/3b30f94f4f129e67be9b6c677aac582923da1fab/src/vs/base/common/observableInternal/index.ts#L22-L25

Reply:
No, as this the first one is for debugging code  written for observables, whereas this is a flag to debug the observable algorithm itself.